index.html

```diff
@@ -20,160 +20,165 @@
   </head>
 
   <body>
-    <div class="header">
+    <section class="header">
```

Comment:
There's a dedicated HTML element called [`<header>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) that is great to use here